### ISSUE TITLE
"From existing Node.js code" is literally unusable

### ISSUE THREAD

**Issue by AskYous** [Microsoft/nodejstools#1594]
##### Expected Behavior 

Run my Node.js project.

##### Actual Behavior 

Errors saying `The specified path does not exist: 'c:\users\yousef shanawany\documents\visual studio 2017\Projects\db-admin\tsconfig.json'	db-admin1	tsc	0	
`

- NTVS Version: 1.3.0.7
- Visual Studio Version:  2015 & 2017
- Node.js Version: v6.9.5

##### Steps to Reproduce
1. Create a solution using "From existing Node.js code"
2. Errors saying it can't find files.

I recorded how to reproduce this here: https://www.youtube.com/watch?v=Uif3aETblQ4.

I get this for both my nodejs projects. The project in the video was [this one](https://github.com/db-admin/db-admin). I also have another project at work. It fails 100% of the time. I can't do anything. This is literally unusable.

**Comment by AskYous** [comment]
Microsoft said it's a bug when the code is in another folder than the project folder:

https://developercommunity.visualstudio.com/content/problem/62921/creating-typescript-project-from-existing-code-doe.html?childToView=65747#comment-65747

**Comment by paulvanbrenk** [comment]
Now that I look at it more this specific problem could be a TypeScript problem... since they seem to do something weird in the build steps.

Can you share the diagnostic build logs? 

`msbuild /fl /flp:verbosity=diag <projectname>` from the developer command prompt.

**Comment by AskYous** [comment]
@paulvanbrenk I don't know where the developer command prompt. I found the visual studio command window from View > Other Windows > Command window but it errored saying `Command "msbuild" is not valid.` But I also found Tools > NodeJS Tools > Diagnostics Info and it had logs. I pasted its output [**here**](https://pastebin.com/pec1P6dB).

**Comment by paulvanbrenk** [comment]
The developer command prompt is in the start menu, not in VS.

**Comment by AskYous** [comment]
The command prompt errored: 

`'msbuild' is not recognized as an internal or external command,
operable program or batch file.`

**Comment by paulvanbrenk** [comment]
It's called this `Developer Command Prompt for VS 2017`, can you post a screenshot?

**Comment by AskYous** [comment]
@paulvanbrenk oooh ok. I thought it was regular command prompt. Here's the output on [Pastebin](https://pastebin.com/ivFqLNmE), and here's a screenshot:

![image](https://user-images.githubusercontent.com/791996/26951413-e138cb76-4c55-11e7-945c-56c885969709.png)

**Comment by paulvanbrenk** [comment]
Cool there should be an `msbuild.log` file generated, can you post that too.

**Comment by AskYous** [comment]
@paulvanbrenk I can't find it in my project folder. Where is it located?

**Comment by paulvanbrenk** [comment]
Possibly in the `c:\windows\system32` folder where you kicked off the build from.

**Comment by AskYous** [comment]
I checked `c:\windows\system32` and it's not there:

![image](https://user-images.githubusercontent.com/791996/26952932-7330cbea-4c5c-11e7-9f1c-22b617c7d3c2.png)

**Comment by paulvanbrenk** [comment]
Weird, can you search `c:\users`

**Comment by AskYous** [comment]
Ok, I found it in `c:\windows\SysWOW64`. [Here](https://gist.github.com/AskYous/86f81db1c1d6bce9f23c7811f3a35158) is the output.

**Comment by paulvanbrenk** [comment]
Thanks, I noticed the generated project file and the sources are on different drives, does this still repro when both are on the same drive?

**Comment by AskYous** [comment]
Yes it does. [Here](https://gist.github.com/AskYous/dc43f56cd2f8b9edbec01c6c06827893) is the output for msbuild.log, and below is a screenshot of the dev command line:

![image](https://user-images.githubusercontent.com/791996/26955926-188c0480-4c6f-11e7-8fbb-fc722c0cd962.png)

**Comment by paulvanbrenk** [comment]
Thanks, this looks like a pretty significant change...

**Comment by AskYous** [comment]
Weird. I get the same error on visual studio though:
![image](https://user-images.githubusercontent.com/791996/26998407-18e8d51a-4d38-11e7-93b7-d4005ec4365b.png)

**Comment by AskYous** [comment]
@paulvanbrenk if it helps, I receive this error message a lot when I open a project:
![image](https://user-images.githubusercontent.com/791996/27048503-f6b2e9f6-4f60-11e7-8962-24914c828944.png)

Though I have TypeScript installed.

**Comment by paulvanbrenk** [comment]
Does any of this happen when you create a new Node Console App?

My suggestion is to create a Node Console App and copy the ts/js files you want to use in your project into that folder, and add it to the project.

**Comment by AskYous** [comment]
@paladique It doesn't happen if I create a new project and copy the files in there. That's what I've been doing so far. It works fine. The bug is only when the files are in an outside seperate folder.
